mods/ts/chrome.yaml

```diff
@@ -1,394 +1,300 @@
-	border-t: 33,28,18,5
-	corner-tl: 28,28,5,5
-	corner-tr: 51,28,5,5
-	corner-bl: 28,51,5,5
-	corner-br: 51,51,5,5
-sidebar-button-observer-disabled: chrome.png
-	background: 173,5,18,18
-	border-r: 191,5,5,18
-	border-l: 168,5,5,18
-	border-b: 173,23,18,5
-	border-t: 173,0,18,5
-	corner-tl: 168,0,5,5
-	corner-tr: 191,0,5,5
-	corner-bl: 168,23,5,5
-	corner-br: 191,23,5,5
-sidebar-button-observer-highlighted-disabled: chrome.png
-	background: 173,5,18,18
-	border-r: 191,5,5,18
-	border-l: 168,5,5,18
-	border-b: 173,23,18,5
-	border-t: 173,0,18,5
-	corner-tl: 168,0,5,5
-	corner-tr: 191,0,5,5
-	corner-bl: 168,23,5,5
-	corner-br: 191,23,5,5
-
-observer-scrollpanel-button: dialog.png
-	background: 769,257,126,126
-	border-r: 895,257,1,126
-	border-l: 768,257,1,126
-	border-b: 769,383,126,1
-	border-t: 769,256,126,1
-	corner-tl: 768,256,1,1
-	corner-tr: 895,256,1,1
-	corner-bl: 768,383,1,1
-	corner-br: 895,383,1,1
-
-observer-scrollpanel-button-hover: dialog.png
-	background: 769,257,126,126
-	border-r: 895,257,1,126
-	border-l: 768,257,1,126
-	border-b: 769,383,126,1
-	border-t: 769,256,126,1
-	corner-tl: 768,256,1,1
-	corner-tr: 895,256,1,1
-	corner-bl: 768,383,1,1
-	corner-br: 895,383,1,1
-
-observer-scrollpanel-button-pressed: dialog.png
-	background: 897,257,126,126
-	border-r: 1023,257,1,126
-	border-l: 896,257,1,126
-	border-b: 897,383,126,1
-	border-t: 897,256,126,1
-	corner-tl: 896,256,1,1
-	corner-tr: 1023,256,1,1
-	corner-bl: 896,383,1,1
-	corner-br: 1023,383,1,1
-
-observer-scrollpanel-button-disabled: dialog.png
-	background: 769,385,126,126
-	border-r: 895,385,1,126
-	border-l: 768,385,1,126
-	border-b: 769,511,126,1
-	border-t: 769,384,126,1
-	corner-tl: 768,384,1,1
-	corner-tr: 895,384,1,1
-	corner-bl: 768,511,1,1
-	corner-br: 895,511,1,1
-
-observer-scrollheader-selected: dialog.png
-	background: 769,385,126,126
-	border-r: 895,385,1,126
-	border-l: 768,385,1,126
-	border-b: 769,511,126,1
-	border-t: 769,384,126,1
-	corner-tl: 768,384,1,1
-	corner-tr: 895,384,1,1
-	corner-bl: 768,511,1,1
-	corner-br: 895,511,1,1
-
-observer-scrollitem-selected: dialog.png
-	background: 897,257,126,126
-	border-r: 1023,257,1,126
-	border-l: 896,257,1,126
-	border-b: 897,383,126,1
-	border-t: 897,256,126,1
-	corner-tl: 896,256,1,1
-	corner-tr: 1023,256,1,1
-	corner-bl: 896,383,1,1
-	corner-br: 1023,383,1,1
-
-observer-scrollitem-hover: dialog.png
-	background: 769,257,126,126
-	border-r: 895,257,1,126
-	border-l: 768,257,1,126
-	border-b: 769,383,126,1
-	border-t: 769,256,126,1
-	corner-tl: 768,256,1,1
-	corner-tr: 895,256,1,1
-	corner-bl: 768,383,1,1
-	corner-br: 895,383,1,1
+	production-tooltip-time: 416,208,16,16
+	production-tooltip-power: 480,160,16,16
+	production-tooltip-cost: 480,144,16,16
+	production-iconoverlay: 168,392,64,48
+	
+scrollbar: buttons.png
+	down_arrow: 16,140,16,16
+	down_pressed: 16,140,16,16
+	up_arrow: 32,140,16,16
+	up_pressed: 32,140,16,16	
+	
+flags: buttons.png
+	gdi: 30,112,30,15
+	nod: 0,112,30,15
+	random: 60,112,30,15
```

Comment:
This has to be `Random` now otherwise the game will crash. See https://github.com/OpenRA/OpenRA/pull/7476/